internal/campaign/campaign.go

```diff
@@ -131,10 +135,13 @@ func readReposTxtFile(filename string) ([]Repo, error) {
 	return repos, nil
 }
 
-func readPrDescriptionFile() (string, string, error) {
-	file, err := os.Open("README.md")
+func readPrDescriptionFile(filename string) (string, string, error) {
+	if filename == "" {
```

Comment:
I have copied this check from the --repos flag, but I'm wondering whether it is really needed (won't it error if the flag has no argument?)